index.html

```diff
@@ -811,6 +814,123 @@ <h3>
+          The [=identity=] can be used by a service that collects lists of web
+          applications to uniquely identify applications.
+        </p>
+        <p class="note">
+          The [=identity=] is processed like a URL but it doesn't point to a
+          resource that can be navigated to, so it's not required to be
+          [=URL/within scope=].
+        </p>
+        <p>
+          To <dfn>process the `id` member</dfn>, given [=object=]
+          |json:JSON|, [=ordered map=] |manifest:ordered map|:
+        </p>
+        <ol class="algorithm">
+          <li>Set |manifest|["id"] to |manifest|["start_url"].
+          </li>
+          <li>If the type of |json|["id"] is not [=string=], return.
+          </li>
+          <li>If |json|["id"] is the empty string, return.
+          </li>
+          <li>Let |base origin| be |manifest|["start_url"]'s [=url/origin=].</li> 
+          <li>Let |id:URL| be the result of [=URL Parser|parsing=] |json|["id"] with  |base origin| as the base URL.
+          </li>
+          <li>If |id| is failure, return.
+          </li>
+          <li>If |id| is not [=same origin=] as |manifest|["start_url"], return.
+          </li>
+          <li>Otherwise, set |manifest|["id"] to |id|.</li>
+        </ol>
+        <aside class="example">
+          <p>Below table shows some example cases of the |identity| processing algorithm.</p>
+          <table class="data">
+            <tr>
+              <th>|json|["id"]</th>
+              <th>|manifest|["start_url"]</th>
+              <th>|manifest|["id"]</th>
+              <th>Explaination</th>
+            </tr>
+            <tr>
+              <td><i>undefined</i></td>
+              <td>"https://example.com/my-app/start"</td>
+              <td>"https://example.com/my-app/start"</td>
+            </tr>
+            <tr>
+              <td>""</td>
+              <td>"https://example.com/my-app/start"</td>
+              <td>"https://example.com/my-app/start"</td>
+            </tr>
+            <tr>
+              <td>"/"</td>
+              <td>"https://example.com/my-app/start"</td>
+              <td>"https://example.com/"</td>
+            </tr>
+            <tr>
+              <td>"foo"</td>
+              <td>"https://example.com/my-app/start"</td>
+              <td>"https://example.com/foo"</td>
+            </tr>
+            <tr>
+              <td>"./foo"</td>
+              <td>"https://example.com/my-app/start"</td>
+              <td>"https://example.com/foo"</td>
+            </tr>
+            <tr>
+              <td>"https://example.com/foo"</td>
+              <td>"https://example.com/my-app/start"</td>
+              <td>"https://example.com/foo"</td>
+            </tr>
+            <tr>
+              <td>"https://anothersite.com/foo"</td>
+              <td>"https://example.com/my-app/start"</td>
+              <td>"https://example.com/my-app/start"</td>
+            </tr>
+            <tr>
+              <td>"%6D"</td>
```

Comment:
I don't think this one is right... try:

```JS
console.log(new URL("%6D", document.location).href)
```

It just gives back "https://whatever/%6D" (i.e., it doesn't do the URL component decoding)

Reply:
hmm, it's working for me?
![Screenshot from 2021-09-22 21-29-52](https://user-images.githubusercontent.com/3324084/134454769-353b9d70-62b7-4b46-a753-8a3a6c1572b3.png)

Reply:
Maybe we should replace this example with a query + fragment example (i.e., query is retained, fragment gets dropped).

Reply:
oh lols no... Firefox and Safari return different results to Chrome on this 🥴

Safari 15: 
![Screen Shot 2021-09-23 at 3 27 20 pm](https://user-images.githubusercontent.com/870154/134458474-cfb75d8e-0a80-4da2-8cbf-6b20d8e27e3f.png)

Firefox 92.0: 
![Screen Shot 2021-09-23 at 3 28 04 pm](https://user-images.githubusercontent.com/870154/134458521-284168f0-14dc-4d60-a716-8cb31deece3a.png)

And yes, you are right... Chrome returns "m".

Would need to check URL spec for which is right.

Reply:
@annevk, sorry, I couldn't figure the above out from reading the URL spec. Are Firefox and Safari right? or is Chrome right in the case above?

Reply:
Firefox and Safari are correct. https://jsdom.github.io/whatwg-url/#url=aHR0cHM6Ly9leGFtcGxlLmNvbS8lNkQ=&base=YWJvdXQ6Ymxhbms= might be useful for such questions.

Reply:
Oh! Nice!!! I'd not seen that tool. Thank you!

Reply:
wow, thanks for finding that! I will update the example and investigate the behavior on Chrome end.

Reply:
Looks like there is a discussion here
https://github.com/whatwg/url/issues/606 

I will remove this example since this is not settled yet.